stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/utils/PairExtensions.kt

```diff
@@ -0,0 +1,16 @@
+package io.getstream.chat.android.offline.utils
+
+import io.getstream.chat.android.core.internal.InternalStreamChatApi
+
+/**
+ * Converts a pair of channelType and channelId into cid.
+ *
+ * @return String CID of the given channel type and id.
+ */
+@Throws(IllegalStateException::class)
```

Comment:
`validateCid` throws `IllegalArgumentException` + let's mention this in KDoc

Reply:
Yeah, I added that based on `checkNotNull`. But as per above comment, it doesn't make sense.